src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -56,9 +59,8 @@ const LightUpChinatownPage = () => {
               major blocks of Chinatown. We hope you will join us.
             </Trans>
           </SummaryBody>
-        </TextContainer>
-        <MapContainer>
-          <Map src={mapImg} /> {/* TODO: Replace with GIF */}
+          <br></br>
+          <br></br>
```

Comment:
You don't need to add the </br> do you?

Reply:
@justintmckibben I think you lost a word or something here